test/e2e/onINP-test.js

```diff
@@ -696,17 +697,18 @@ describe('onINP()', async function () {
       if (!browserSupportsINP) this.skip();
 
       await navigateTo(
-        '/test/inp?click=100&attribution=1&doubleCall=1&generateTarget2=1',
-        {
-          readyState: 'complete',
-        },
+        '/test/inp?click=150&attribution=1&doubleCall=1&generateTarget2=1' +
+          '&reportAllChanges=1&reportAllChanges2=1',
       );
 
+      // Wait until the library is loaded and the first paint occurs to ensure
+      // The 40ms event duration is set
+      await webVitalsLoaded();
+      await firstContentfulPaint();
+
       const h1 = await $('h1');
       await simulateUserLikeClick(h1);
 
-      await stubVisibilityChange('hidden');
-
```

Comment:
@philipwalton this was weird and this change shouldn't be necessary.

Firefox would often fail to report only 1 (and sometimes none!) with this method. But changing it to `&reportAllChanges=1&reportAllChanges2=1` fixed it.

Since the point of this test wasn't to test the visibilitychange thing I was OK changing it, but it might warrant further investigation to see if there's some other flaw in multiple `onINP` calls in Firefox not playing nice with each other (e.g. the reporting of the first `onINP` prevents the second from getting the callback).

Reply:
SGTM